tests/unit_tests/test_material.py

```diff
@@ -32,6 +32,7 @@ def test_add_components():
                   'O16': 1.0,
                   'Zr': 1.0,
                   'O': 1.0,
+                  'Ag110m': 1.0,
```

Comment:
Our format for metastables is as follows:
```suggestion
                  'Ag110_m1': 1.0,
```